### PR TITLE
chore(demo): update disable-animation page

### PR DESCRIPTION
**Pull request by splincode**
## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Build or CI related changes
- [ ] Tests related changes
- [ ] Documentation content changes

## What is the current behaviour?

Closes #

### CODE REVIEW

File: projects/demo/src/modules/info/testing/disable-animation/disable-animation.template.html [hunk 1]
```diff
@@ -15,32 +12,65 @@
...
-            target="_blank"
-            tuiLink
-        >
-            Cypress
-        </a>
-        for screenshot testing of our ui-kit components.
+        <code>&#64;taiga-ui/cdk</code>
+        used
+        <ng-container *ngIf="'TUI_REDUCED_MOTION' | tuiDocTypeReference as refs">
+            <code *ngFor="let ref of refs">
```
Comment (waterplea) [inline projects/demo/src/modules/info/testing/disable-animation/disable-animation.template.html:18]: *ngFor="let ref of 'TUI_REDUCED_MOTION' | tuiDocTypeReference"

File: projects/demo/src/modules/info/testing/disable-animation/disable-animation.component.ts [hunk 2]
```diff
@@ -3,8 +3,10 @@ import {ChangeDetectionStrategy, Component} from '@angular/core';
 @Component({
     selector: 'disable-animation',
     templateUrl: './disable-animation.template.html',
+    styleUrls: ['../../../../modules/app/home/home.style.less'],
```
Comment (waterplea) [inline projects/demo/src/modules/info/testing/disable-animation/disable-animation.component.ts:6]: I think we should either copy-paste or create a component with those styles, I don't like the idea of entanglement of 2 separate pages with the styles.

**Review by waterplea — APPROVED** [review]

File: projects/demo/src/modules/info/testing/disable-animation/disable-animation.template.html (hunk 1 above)
Comment (splincode) [reply projects/demo/src/modules/info/testing/disable-animation/disable-animation.template.html]: fixed

File: projects/demo/src/modules/info/testing/disable-animation/disable-animation.component.ts (hunk 2 above)
Comment (splincode) [reply projects/demo/src/modules/info/testing/disable-animation/disable-animation.component.ts]: fixed

**Review by vladimirpotekhin — APPROVED** [review]

**Pull request merged**